### DISCUSSION TITLE
The start of things

### DISCUSSION THREAD

**Discussion by DeeEmm** [DeeEmm/DIY-Dyno#3] [Build Threads]
It's been a few years since I originally started this project, in fact it was back in 2016 that I originally had the idea to build an engine dyno. An initial burst of enthusiasm allowed me to create a basic engine run stand that I was then gong to add a brake to but life got in the way and that's kind of how far it got.

![IMG_4610](https://user-images.githubusercontent.com/3038710/235894981-33c700e2-ed18-4c63-ac66-0e611d7a9fd8.jpg)

![IMG_4609](https://user-images.githubusercontent.com/3038710/235896710-fa79bd6a-2be6-4e12-9bd6-0b2554c55176.jpg)

The dyno is only for testing Aircooled VW engines and so is specifically designed with that purpose in mind. It uses a Kombi bell-housing to mount the engine, much as it would be in a vehicle. The bell-housing also holds the starter motor.

The intention was to use a hydraulic motor and metering valve as the brake. With a hydraulic tank mounted underneath the trolley along with a fan assisted oil cooler. This was just based on stuff that I managed to pick up cheaply at the time. It's probably not the optimal way to construct a brake dyno, but it meets the budget constraints.

As well as the physical dyno itself, the other major aspect of the project was to develop a software based solution for dyno control and data acquisition. Back then I think I was planning to use an Arduino, or perhaps even a RaspberryPi but having developed a number of recent projects on the ESP32 platform, this has now changed.

System architecture will be based on the ESP32 and custom shield that will support the necessary electronics. All of this is essentially TBD but I will have a basic outline hopefully pretty soon.

**Comment by DeeEmm** [comment]
So I managed to pick up an unfinished water brake. I had seen this project posted on the Facebook group a while back and was surprised to see it posted for sale. The fella who made it had ended up buying an off the shelf water brake for his business and so never had the need to finish it. I was lucky enough to to be able to get first in line and after sorting out the sale have now taken ownership of a mostly finished water brake.

![350524575_1272983553590829_5563121142104746398_n](https://github.com/DeeEmm/DIY-Dyno/assets/3038710/4fb881eb-84d1-4cea-bb12-e58602f94baa)

![350527755_562057752675606_649187970876947947_n](https://github.com/DeeEmm/DIY-Dyno/assets/3038710/5226ee0c-b763-4a0a-8361-05480c65cf61)

![350542910_950708332640420_9061544106362872143_n](https://github.com/DeeEmm/DIY-Dyno/assets/3038710/ec9c62a7-49ed-4a36-aa0b-49ac18ffa6b4)

![350768297_927429011853053_7034926228661912768_n](https://github.com/DeeEmm/DIY-Dyno/assets/3038710/333451aa-9e92-40a1-b3fa-5e6da9df6d6b)

There is some machining to do to finish the brake off which is mostly just machining the flow and return ports and also setting up the rotor clearances. I was hoping to get into this after I received it, but have been stuck doing house renovations which have taken up a lot more of my time than I would have liked.

Moving from a hydraulic based brake to a water brake is a large change of direction for me and so I now need to get my head around the infrastructure required to support a water brake. What once involved a hydraulic tank and an oil cooler now needs some kind of water tank and water cooling system.

I think realistically it's going to be the new year before I can get stuck into this, but the plan is to chip away at the code in the mean time.

I also had a message from a follower on Discord who is interested in contributing, so hopefully we can get some movement on the code side of the project. I definitely get more motivation when there's some activity going on.

I have made a start on porting the DIY Flow Bench code over, but it is not in a state that I can upload it yet. But what I have done is create some unit tests that anyone can pick up and work on. I figure that if there are contributors that just want to work on something simple like reading date from a load cell, then creating a simple unit test is the easiest thing to do.

**Reply by Ronnie180** [reply]
This is a nice looking water brake Mick, Looks alot like the Land & Sea Dynomite. Mine is just hooked up to an inside water faucet that has 50-60 lbs of pressure & outlet just goes to drain so its consistent temp all the time, Yeah not the cheapest or most efficient but has always been consistent with back to back pulls.  I know a guy that used a 30 gal tank, hung it from the ceiling about 3 foot above the dyno & for the cooling he just took a 50 foot roll of copper refrigeration tubing uncoiled it just enough to have space between the coil rounds, mounted it to a piece of aluminum that had slots 1 inch wide cut all in it, mounted that to a frame & used a small squirrel cage out of a small gas furnace for the cooling

Then another went all out with a tank, pump with bypass, pressure switch & a 4 coil aluminum racing radiator & a 2 fan 12 volt radiator fan for cooling. So tons of ways to get it going for sure.

**Reply by DeeEmm** [reply]
Yeah he's done a really nice job and I agree very similar looking to the L&S units.

The water faucet might work well for me. The Dyno is not something that will be in constant use so for the small amount of use it will see there will likely not be too much wastage. In fact I would think that it might even use less water than having to replenish something like an IBC where the water had mostly evaporated away between uses. Plus I can use the water on the veggie garden or to top up the pond.

The omission of the additional infrastructure definitely gets me up and running quicker and saves on a bit of real estate in my workshop, which is at a bit of a premium.

**Reply by Ronnie180** [reply]
Yeah, my Stuska is 9 inches in diameter & with the 60 lbs faucet pressure I can choke a 50 hp 2 stroke out & water usage I can make 3 back to back pulls from warm up with slight load to finish on 5-6 gallons of water so not really as much as it would seem to be. 

I agree on probably save water this way instead of recycle unless you had some way to totally seal the system which is probably not going to happen. Big hp engines I could see a recycle system more than likely being the best since you moving way more water. The one I mentioned that went all out, he did have a leveling sensor & fill valve for his tank.

**Reply by DeeEmm** [reply]
I'm probably around 100-150hp with most of the stuff I will test so not massive HP-wise.

**Reply by Ronnie180** [reply]
Yeah, you may be okay with 60-70 lbs of pressure on them. If not maybe play with the outlet Orifice, I think mine is 3/8 inch outlet & I'M pretty sure it's the same inlet to. 2 inlets & 1 outlet, plus took me awhile to fine tune the outlet to get it to response pretty quick with the hand valve which is where automation would come in handy lol.

**Reply by DeeEmm** [reply]
Yeah I need to get some more data on the port design for the L&S units and compare before I get stuck in. It seems to be the closest design-wise. I did receive a Fusion360 model of the unit from the guy who made it, which I assume was used to generate the toolpaths for the machining. I need to take a closer look at it and see if there's any port details in the model. 

I did find this image online of an L&S dyno which shows how the outlet port is blended into to final pocket.

![landandsea_small_0088_1](https://github.com/DeeEmm/DIY-Dyno/assets/3038710/beaf4a32-780b-4161-bb87-d95747af99b7)

I expect that I will most likely end up doing this by hand as I'm not sure that there is enough clearance at the required tool approach angle. I still need to check this. The L&S dyno is three piece and has a seperate 'spacer' between the outside parts of the housing, whereas the housing I have is two piece and so the outer wall of the housing shrouds the access. It's not an insurmountable problem, but does mean that I most likely cannot just chuck it on the mill and machine the port out. A bit of time with a die grinder will most likely get it done.

The other option is to machine in at an angle from the outside and make a flanged takeoff plate for the outlet connection. This is probably the most elegant solution but a bit more work with setting the job up to ensure that the hole breaks through in the right spot. 

We recently had the water supply on our property upgraded so I'm hoping that there will be enough flow.
